interp/interp.py

```diff
@@ -0,0 +1,12 @@
+from machine import mem32
+
+# initialise lane 0 on interp: set that we are using all 32 bits
+mem32[0xD0000000 | 0xAC] = 0x1F << 10
```

Comment:
It's not up to me whether this PR gets merged or not, but IMHO it'd be **much** easier to read if you had
```python
SIO_BASE = 0xD0000000

INTERP0_ACCUM0 = 0x80
INTERP0_BASE0 = 0x88
INTERP0_POP_LANE0 = 0x94
INTERP0_CTRL_LANE0 = 0xAC
```
at the top of the file, and then this line could become:
```python
mem32[SIO_BASE + INTERP0_CTRL_LANE0] = 0x1F << 10
```
(note that I've also changed the `|` to a `+`, which is _probably_ more readable to most Python programmers?)

Reply:
Excellent suggestion, done.